examples/tests/sdk_tests/util/env.test.w

```diff
@@ -5,7 +5,7 @@ let NOT_ACTUAL_ENV = "__NOT_ACTUAL_ENV_SHOULD_FAIL__";
 /**
  * env()
  */
-assert(util.env("PATH").length > 0);
+assert(util.env("WING_TARGET") != "");
```

Comment:
Using a worker thread broke this on windows. Not sure why, but the variables below should also come from the CLI process so I'm not too concerned about a huge regression here.

Reply:
Not sure I understand. Wouldn't we expect `PATH` to be defined?

Also/ why worker threads?